src/multipart.jl

```diff
@@ -53,7 +53,7 @@ function Base.read(f::Form, n::Integer)
     return result
 end
 
-function Form(d::Dict)
+function Form(d::Vector{Pair{String,T}}) where T<:Any
```

Comment:
Maybe just allow any type of `d` ?

That way you can pass in anything that makes sense with : `for (i, (k, v)) in enumerate(d)`
e.g. an iterator that produces Pairs, or Tuples, or a Dict ...